core/src/test/scala/org/apache/spark/deploy/master/MasterSuiteBase.scala

```diff
@@ -49,7 +49,7 @@ object MockWorker {
 class MockWorker(master: RpcEndpointRef, conf: SparkConf = new SparkConf) extends RpcEndpoint {
   val seq = MockWorker.counter.incrementAndGet()
   val id = seq.toString
-  override val rpcEnv: RpcEnv = RpcEnv.create("worker", "localhost", seq,
+  override val rpcEnv: RpcEnv = RpcEnv.create("worker", "localhost", 0,
```

Comment:
just for my education, port 0 means using any available port?

Reply:
if so can we add a comment to mention it?

Reply:
> if so can we add a comment to mention it?

Added.